shuttle/views.py

```diff
@@ -9,6 +9,10 @@ class View(FlaskView):
     def index(self):
         return render_template('index.html')
 
+    @route('/request-shuttle')
+    def request(self):
+        return render_template('request_shuttle.html')
+
```

Comment:
For the time being you could probably throw all things involving student interaction into the /schedules init.